plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py

```diff
@@ -77,246 +85,98 @@ def cluster_yaml() -> str:
     # Set integration test cluster name
     ray_cluster_cfg.cluster_name = cluster_name
     # Delete cluster once integration tests are done.
-    ray_cluster_cfg.provider.cache_stopped_nodes = False
+    ray_cluster_cfg.provider.cache_stopped_nodes = True
     with tempfile.NamedTemporaryFile(suffix=".yaml", delete=False) as f:
         with open(f.name, "w") as file:
-            print(
-                ray_cluster_cfg.pretty(), file=file,
-            )
+            print(ray_cluster_cfg.pretty(), file=file)
         ray_up(f.name)
+        global temp_yaml
+        temp_yaml = f.name
         yield f.name
         ray_down(f.name, ray_cluster_cfg)
 
 
+@pytest.fixture(scope="module")
+def remote_dir() -> None:
+    global temp_yaml
+    global temp_remote_dir
+    global remote_working_dir
+    remote_working_dir = ray_working_dir(temp_yaml, False)
+    with ray_tmp_dir(temp_yaml, False) as dir:
+        temp_remote_dir = dir
+        yield
+        print("Clean up remote tmp dir.")
+
+
 @pytest.mark.skipif(sys.platform.startswith("win"), reason=win_msg)
 @pytest.mark.usefixtures("cluster_yaml")
 @pytest.mark.parametrize(
     "launcher_name, overrides",
     [
         (
             "ray_aws",
-            [f"hydra.launcher.params.ray_cluster_cfg.cluster_name={cluster_name}"],
+            [
+                f"hydra.launcher.params.ray_cluster_cfg.cluster_name={cluster_name}",
+                "hydra.launcher.params.stop_cluster=False",
+                "hydra.launcher.params.download_log=True",
+            ],
         )
     ],
 )
-@pytest.mark.parametrize("sweep_runner", [True], indirect=True)
-class TestRayLauncher(LauncherTestSuite):
+@pytest.mark.parametrize("sweep_runner", [sweep_dir], indirect=True)
+class TestRayAWSLauncher(LauncherTestSuite):
     """
     Run the Launcher test suite on this launcher.
     """
 
-    pass
+    # pass in functions from within hydra_ray_launcher when pytest causes pickling issue with Ray.
+    def use_task_function(self) -> bool:
+        return False
 
 
-# This mimics what's being tested in IntegrationTestSuite but adapts to a remote setting.
-# we are not running test_get_orig_dir_multirun because of the remote setting.
+@pytest.mark.usefixtures("remote_dir")
+@pytest.mark.usefixtures("cluster_yaml")
```

Comment:
the fixture names are opaque to me.
I prefer fixture names to be a verb, like "clear_singletons". it makes it easier to understand what they are for.